### PR TITLE
data: SreejitSengupta

### PR DESCRIPTION
**Pull request by Sreejit-Sengupto**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/sreejit-sengupto.json [hunk 1]
```diff
@@ -1,26 +1,72 @@
 {
-  "name": "SREEJIT SENGUPTA",
-  "bio": "Open Source Enthusiast! On and into Web Development. Aspiring Open Sorcerer. Constantly trying to Upskill myself. Learning everyday!",
+  "name": "Sreejit Sengupta",
+  {
```
Comment (sital002) [inline data/sreejit-sengupto.json:3]:
```suggestion
  
```

File: data/sreejit-sengupto.json [hunk 2]
```diff
@@ -1,26 +1,72 @@
...
+  "bio": "JavaScript | React | Frontend Developer | Tailwind | Blogger at **Hashnode**"
+  },
```
Comment (sital002) [inline data/sreejit-sengupto.json:5]:
```suggestion
  ,
```

File: data/sreejit-sengupto.json [hunk 3]
```diff
@@ -1,26 +1,72 @@
...
+{
```
Comment (sital002) [inline data/sreejit-sengupto.json:6]:
```suggestion

```

File: data/sreejit-sengupto.json [hunk 4]
```diff
@@ -1,26 +1,72 @@
...
+  "tags": ["Blogger", "Javascript", "Tailwind", "React", "Tweeter"]
+},
```
Comment (sital002) [inline data/sreejit-sengupto.json:8]:
```suggestion
,
```

File: data/sreejit-sengupto.json [hunk 5]
```diff
@@ -1,26 +1,72 @@
...
+      "description": "Wrote my first blog describing three ways to include CSS in an HTML file",
+      "url": "sreesen.hashnode.dev"
+    },
+    {
+      "title": "Cloned Battlegrounds Mobile India's website",
+      "date": "June 24, 2023",
+      "icon": "FaGamepad",
+      "color": "grey",
+      "description": "Cloned the entire website of BGMI using React, React-Router and TailwindCSS"
+      "url": "https://battlegrounds.vercel.app/"
```
Comment (sital002) [inline data/sreejit-sengupto.json:53]:
```suggestion
      "url": "https://battlegrounds.vercel.app/",
```
There is a comma required here

**Review by sital002 — CHANGES_REQUESTED** [review]
Your filename are username arenot matching Please rename your filename from ```sreejit-sengupto.json``` to ```Sreejit-Sreejit.json``` I woul recoomwd you to watch our quick start guide
https://www.biodrop.io/docs/quickstart-json (especially section 7).

File: data/sreejit-sengupto.json [hunk 6]
```diff
@@ -1,26 +1,72 @@
...
+      "color": "grey",
+      "description": "Wrote my first blog describing three ways to include CSS in an HTML file",
+      "url": "sreesen.hashnode.dev"
+    },
+    {
+      "title": "Cloned Battlegrounds Mobile India's website",
+      "date": "June 24, 2023",
+      "icon": "FaGamepad",
+      "color": "grey",
+      "description": "Cloned the entire website of BGMI using React, React-Router and TailwindCSS"
```
Comment (sital002) [inline data/sreejit-sengupto.json:52]:
```suggestion
      "description": "Cloned the entire website of BGMI using React, React-Router and TailwindCSS",
```
There is a comma missing.

**Comment by Sreejit-Sengupto** [comment]
I have made the required changes

**Review by sital002 — APPROVED** [review]
Looks good Thanks 🎉🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**